### ISSUE TITLE
Gallery thumbs should use flexbox instead of fixed width to consume space

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-asset-admin#566]
When we started to build the files area for SS4 we hadn't decided on using flexbox yet so we used a fixed width for thumbnails. Now that we are using flexbox throughout the files area we should update the thumbs to use flexbox to evenly space and spread thumbs to the available space.

Example: https://www.taniarascia.com/how-to-build-a-responsive-image-gallery-with-flexbox/

**Comment by chillu** [comment]
Agreed, we should use flexbox pretty much everywhere. For me that's more of a long-term maintenance concern though. I'm happy to live with with the visual spacing issues for now. Removing from milestone

![image](https://user-images.githubusercontent.com/111025/29499533-ad1eb0c0-8667-11e7-8172-0124e5cbb038.png)

**Comment by clarkepaul** [comment]
FYI We've had someone from the team do a quick look into this and it wasn't an easy one to do apparently, without involving JS.
